tests/components/google_assistant/test_trait.py

```diff
@@ -507,7 +507,7 @@
             climate.ATTR_CURRENT_HUMIDITY: 25,
             climate.ATTR_OPERATION_MODE: climate.STATE_AUTO,
             climate.ATTR_OPERATION_LIST: [
-                climate.STATE_OFF,
+                climate.STATE_IDLE,
```

Comment:
So I thought more about this and this is wrong.

Google Assistant queries for the operation mode, not the current operation. 

https://developers.google.com/actions/smarthome/traits/temperaturesetting

Reply:
It should be STATE_OFF.

Reply:
That makes sense. Unfortunately more components use `climate.STATE_IDLE` than `STATE_OFF` at the moment. Should I wait for https://github.com/home-assistant/architecture/issues/22 to complete or just fix this in the short term? I think a major rewrite will miss the next release window.

Reply:
Google Assistant is already correct, so we should stick with that. Because if we mess this up, it means that people will have to say different things. We should go fix the platforms.